web/src/app/admin/admin-alert-counts/AlertCountTable.tsx

```diff
@@ -0,0 +1,168 @@
+        )
+      }
+      return ''
+    },
+  },
+  {
+    field: 'total',
+    headerName: 'Total',
+    type: 'number',
+    width: 150,
+    valueGetter: (params: GridValueGetterParams) => params.row.total,
+  },
+  {
+    field: 'max',
+    headerName: 'Max',
+    type: 'number',
+    width: 150,
+    valueGetter: (params: GridValueGetterParams) => params.row.max,
+  },
+  {
+    field: 'avg',
+    headerName: 'Average',
+    type: 'number',
+    width: 150,
+    valueGetter: (params: GridValueGetterParams) => params.row.avg,
+  },
+  {
+    field: 'serviceID',
+    headerName: 'Service ID',
+    valueGetter: (params: GridValueGetterParams) => {
+      return params.row.id || ''
+    },
+    hide: true,
+    width: 300,
+  },
+]
+
+export default function AlertCountTable(
+  props: AlertCountTableProps,
+): JSX.Element {
+  const classes = useStyles()
+  const alertCounts = useMemo(() => props.alertCounts, [props.alertCounts])
+
+  const csvOpts = useMemo(
+    () => ({
+      urlPrefix: location.origin + pathPrefix,
+      alertCounts: alertCounts,
+    }),
+    [props.alertCounts],
+  )
+  const csvData = useWorker('useAlertCountCSV', csvOpts, '')
+  const link = useMemo(
+    () => URL.createObjectURL(new Blob([csvData], { type: 'text/csv' })),
+    [csvData],
+  )
+
+  function CustomToolbar(): JSX.Element {
+    const apiRef = useGridApiContext()
+    const currentPage = gridPaginatedVisibleSortedGridRowEntriesSelector(
+      apiRef,
+    ).map((page) => ({
+      serviceName: page.model.serviceName,
+      dailyCounts: page.model.dailyCounts,
+      id: page.model.id,
+      total: page.model.total,
+      max: page.model.max,
+      avg: page.model.avg,
+    }))
+
+    useEffect(() => {
+      // only set graphData if currentPage changes
+      if (JSON.stringify(props.graphData) !== JSON.stringify(currentPage)) {
+        props.setGraphData(currentPage)
+      }
+    }, [currentPage])
+
+    return (
+      <GridToolbarContainer className={gridClasses.toolbarContainer}>
+        <Grid container justifyContent='space-between'>
+          <Grid item>
+            <GridToolbarColumnsButton />
+            <GridToolbarFilterButton />
+            <GridToolbarDensitySelector />
+          </Grid>
+          <Grid item>
+            <AppLink
+              to={link}
+              download={`all-services-alert-counts-${props.startTime}-to-${props.endTime}.csv`}
+            >
+              <Button
+                size='small'
+                startIcon={<DownloadIcon fontSize='small' />}
+              >
+                Export
+              </Button>
+            </AppLink>
+          </Grid>
+        </Grid>
+      </GridToolbarContainer>
+    )
+  }
+
+  return (
+    <Grid container className={classes.tableContent}>
+      <Grid item xs={12} data-cy='alert-count-table'>
+        <DataGrid
+          rows={props.alertCounts ?? []}
+          loading={props.loading}
+          pageSize={7}
```

Comment:
i'm seeing this warning in the console, are we intending to keep the data at 7 per page without other options?

```
The page size `7` is not preset in the `rowsPerPageOptions`
Add it to show the pagination select.
```

Reply:
Yep, keeping the default at 7, but forgot the ``rowsPerPageOptions={[7]}`` is still required.